tools/nimgrep.nim

```diff
@@ -62,10 +62,11 @@ Options:
        -s[:asc|desc]  - ascending (default: recent files go last) or descending
   --group, -g         group matches by file
   --newLine, -l       display every matching line starting from a new line
-  --limit[:N], -m[:N] limit max width of lines from files by N characters (80)
-  --fit               calculate --limit from terminal width for every line
-  --onlyAscii, -o     use only printable ASCII Latin characters 0x20-0x7E
-                      (substitutions: 0 -> @, 1-0x1F -> A-_, 0x7F-0xFF -> !)
+  --cols[:N]          limit max width of lines from files by N characters (off)
+  --cols:auto, -%     calculate columns from terminal width for every line
```

Comment:
the problem with abbreviations like `-%` and `-@` is they're cryptic, hard to search, non standard/self documenting (and -%` might cause escaping problems on some shells, not sure)

same comment as araq's comment in my PR: https://github.com/nim-lang/Nim/pull/15687#discussion_r511829251
> Please only introduce the long version, I cannot remember all these single char abbreviations anymore.

It's easy enough to alias your own commands privately in your shell with your favorite options.

That said, this isn't a blocker in case you insist on keeping those.

Reply:
all fair, but I left them, see my comment below.